maven-release-plugin/src/it/projects/prepare/MRELEASE-1053/module-a/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project>
+  <parent>
+    <groupId>org.apache.maven.plugin.release</groupId>
+    <artifactId>mrelease-1053-parent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <relativePath>../module-parent/pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.maven.plugin.release</groupId>
+  <artifactId>module-a</artifactId>
+  <name>module-a</name>
+  <version>1.0-SNAPSHOT</version>
+  <url>http://maven.apache.org</url>
```

Comment:
remove url

Reply:
Fixed in https://github.com/apache/maven-release/commit/42eeaf70dafedc759e7bd5421842d2ad687f7deb.